libs/core/src/lib/busy-indicator/busy-indicator-extended/busy-indicator-extended.directive.ts

```diff
@@ -0,0 +1,27 @@
+import { AfterContentInit, Directive, ElementRef } from '@angular/core';
+
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[fd-busy-indicator-extended]'
+})
+export class BusyIndicatorExtendedDirective implements AfterContentInit {
+    /** @hidden */
+    constructor(private _eleRef: ElementRef) {}
+    ngAfterContentInit(): void {
+        this._appendCssToParent();
+    }
+
+    /** @hidden */
+    _appendCssToParent(): void {
+        const hasLabel = this.elementRef().nativeElement.firstElementChild.getAttribute('label');
+        if (hasLabel !== null) {
+            this.elementRef().nativeElement.parentElement.classList.add('fd-busy-indicator-extended--message-toast');
+            this.elementRef().nativeElement.parentElement.classList.add('fd-busy-indicator-extended');
+        }
+    }
+
+    /** @hidden */
+    elementRef(): ElementRef<HTMLElement> {
```

Comment:
ordering of methods should be 1. lifecycle methods, 2. public, 3. private. So this should go before `_appendCssToParent`